pep-0557.rst

```diff
@@ -426,19 +426,47 @@ field an initial value.
 Mutable default values
 ----------------------
 
-Python stores the default field values in class attributes.
+Python stores default member variable values in class attributes.
 Consider this example, not using Data Classes::
 
   class C:
-      def __init__(self, x=[]):
+      x = []
+      def add(self, element):
+          self.x += element
+
+  o1 = C()
+  o2 = C()
+  o1.add(1)
+  o2.add(2)
+  assert o1.add == [1, 2]
```

Comment:
This should be ``o1.x == [1, 2]``

Reply:
I thought I fixed that. I guess I fixed it in the REPL, but not the PEP code. Thanks.